bouncer/__init__.py

```diff
@@ -27,6 +27,7 @@ def settings():
                                              "9200"),
         "hypothesis_url": os.environ.get("HYPOTHESIS_URL",
                                          "https://hypothes.is"),
+        "sentry_dsn": os.environ.get("SENTRY_DSN"),
```

Comment:
This isn't needed AIUI. Raven is already capable of reading the setting from the environment.